csharp_dotnetcore/8.Suggested-Actions/Suggested_Actions/appsettings.json

```diff
@@ -0,0 +1,4 @@
+﻿{  
+  "MicrosoftAppId": "",
+  "MicrosoftAppPassword": ""
```

Comment:
These should be moved to the .bot file. The only content in appsettings should be botFilePath and botFileSecret.

Reply:
Roger that